documentation/src/main/asciidoc/stories/assembly_wildfly_modules.adoc

```diff
@@ -0,0 +1,21 @@
+[id='ispn_modules']
+:context: server
+= {brandname} Modules for {wildflybrandname}
+To use {brandname} inside applications deployed to {wildflybrandname}, you should install {brandname} modules that:
+
+* Let you deploy applications without packaging {brandname} JAR files within your WAR, EAR, and so on.
```

Comment:
"and so on" seems very casual.

Reply:
@ryanemerson good point. tbh I was leaving it a bit open but it'll only be a WAR or EAR realistically. I guess we could go even more generic and say something like "without packaging Infinispan JAR files in your distribution." But perhaps that is less clear?

Reply:
I think that adds ambiguity tbh.